datasets/monash_tsf/README.md

```diff
@@ -0,0 +1,234 @@
+| fred_md                       | Economic  | 107           | 1M     | 12         | [McCracken and Ng, 2016](https://doi.org/10.1080/07350015.2015.1086655)                                                             |
+| traffic_hourly                | Transport | 862           | 1H     | 48         | [Caltrans, 2020](http://pems.dot.ca.gov/)                                                                                           |
+| traffic_weekly                | Transport | 862           | 1W-WED | 8          | [Caltrans, 2020](http://pems.dot.ca.gov/)                                                                                           |
+| hospital                      | Health    | 767           | 1M     | 12         | [Hyndman, 2015](https://cran.r-project.org/web/packages/expsmooth/)                                                                 |
+| covid_deaths                  | Health    | 266           | 1D     | 30         | [Johns Hopkins University, 2020](https://github.com/CSSEGISandData/COVID-19)                                                        |
+| sunspot                       | Nature    | 1             | 1D     | 30         | [Sunspot, 2015](http://www.sidc.be/silso/newdataset)                                                                                |
+| saugeenday                    | Nature    | 1             | 1D     | 30         | [McLeod and Gweon, 2013](http://www.jenvstat.org/v04/i11)                                                                           |
+| us_births                     | Health    | 1             | 1D     | 30         | [Pruim et al., 2020](https://cran.r-project.org/web/packages/mosaicData)                                                            |
+| solar_4_seconds               | Energy    | 1             | 4S     | 60         | [Godahewa et al. 2021](https://openreview.net/pdf?id=wEc1mgAjU- )                                                                   |
+| wind_4_seconds                | Energy    | 1             | 4S     | 60         | [Godahewa et al. 2021](https://openreview.net/pdf?id=wEc1mgAjU- )                                                                   |
+| rideshare                     | Transport | 2304          | 1H     | 48         | [Godahewa et al. 2021](https://openreview.net/pdf?id=wEc1mgAjU- )                                                                   |
+| oikolab_weather               | Nature    | 8             | 1H     | 48         | [Oikolab](https://oikolab.com/)                                                                                                     |
+| temperature_rain              | Nature    | 32072         | 1D     | 30         | [Godahewa et al. 2021](https://openreview.net/pdf?id=wEc1mgAjU- )      
+
+
+### Dataset Usage
+
+To load a particular dataset just specify its name from the table above e.g.:
+
+```python
+load_dataset("monash_tsf", "nn5_daily")
+```
```

Comment:
The user can pass a custom `prediction_length` or `rolling_evaluations`, so we should list and describe these args here, no?

Reply:
no not in this dataset as those are fixed as well (from the monash side and their preprocessing)

Reply:
Feel free to mention it in the dataset card :)